frontend/src/themes/components/alert.ts

```diff
@@ -40,13 +40,15 @@ const Alert = {
     },
     successToast: {
       container: {
+        margin: "4",
```

Comment:
do we want this margin thing in `baseStyle` so it applies to all variants? not sure what the other two are tho / where it's used

Reply:
toast uses alert under the hood, so i don't think so? bc we don't want to apply the margin to other non-toast alerts (i.e form error messages). let me know if i'm missing something b/c i am not on it today

Reply:
ok ya that makes sense